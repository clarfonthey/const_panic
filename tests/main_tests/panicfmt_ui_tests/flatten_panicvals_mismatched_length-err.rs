use const_panic::{FmtArg, PanicVal, flatten_panicvals};

const _: [PanicVal<'_>; 20] = flatten_panicvals!(FmtArg::DEBUG; 10u8, 20u8);

fn main() {}