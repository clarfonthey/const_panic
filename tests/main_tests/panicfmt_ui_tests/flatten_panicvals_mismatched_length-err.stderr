error[E0308]: mismatched types
 --> tests/main_tests/panicfmt_ui_tests/flatten_panicvals_mismatched_length-err.rs:3:31
  |
3 | const _: [PanicVal<'_>; 20] = flatten_panicvals!(FmtArg::DEBUG; 10u8, 20u8);
  |                         --    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected an array with a size of 20, found one with a size of 2
  |                         |
  |                         help: consider specifying the actual array length: `2`
  |
  = note: expected array `[PanicVal<'static>; 20]`
             found array `[PanicVal<'_>; 2]`
  = note: this error originates in the macro `$crate::__to_pvf_inner` which comes from the expansion of the macro `flatten_panicvals` (in Nightly builds, run with -Z macro-backtrace for more info)
